app/src/main/kotlin/com/mapbox/vision/teaser/replayer/ArReplayNavigationActivity.kt

```diff
@@ -195,7 +195,11 @@ class ArReplayNavigationActivity : AppCompatActivity(), MapboxMap.OnMapClickList
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_ar_navigation_replayer)
 
-        sessionPath = intent.getStringExtra(KEY_SESSION_PATH)
+        sessionPath = intent.getStringExtra(KEY_SESSION_PATH) ?: ""
+        if (sessionPath.isEmpty()) {
```

Comment:
Just `intent.getStringExtra(KEY_SESSION_PATH).isNullOrEmpty` ?

Reply:
session path is used in other places and lateinit, seems this assignment is ok to avoid other boilerplate code